department-of-veterans-affairs/vets-api
# VI-447: Update Identity Dashboard service account scopes

## Summary

This PR updates the Identity Dashboard service account configuration to include the scope for the updating service account configs via the API.

## Related issue(s)

https://jira.devops.va.gov/browse/VI-447

## Testing done

* Run the seeds file.
* Verify the scope has been updated.

## Screenshots

Not relevant.

## What areas of the site does it impact?

Identity Dashboard service account token scopes.

## Acceptance criteria


## Requested Feedback

None.
